akka-http/src/main/scala/akka/http/javadsl/server/RoutingJavaMapping.scala

```diff
@@ -20,6 +21,7 @@ import scala.collection.immutable
 /**
  * INTERNAL API
  */
+@InternalApi
 private[http] object RoutingJavaMapping {
```

Comment:
Currently there is no `impl` package under `akka-http/src/main/scala/akka/http/`. Should I add it?